apps/front-end/src/onest/scholarship/AutomatedForm.jsx

```
@@ -431,52 +378,55 @@ const AutomatedForm = () => {
-            searchForm(formUrl);
-            //   } else if (curr == max) {
-            //     setLoading(true);
-            //     confirmDetails();
-            //   }
-            setLoading(false);
-
-            // } else {
-            //   confirmDetails(formData);
-          }
+        } else if (
+          data?.responses[0]?.message?.order?.items[0].hasOwnProperty("xinput")
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)